x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_settings_collector.js

```diff
@@ -6,24 +6,48 @@
 
 import { get } from 'lodash';
 import { XPACK_DEFAULT_ADMIN_EMAIL_UI_SETTING } from '../../../../../server/lib/constants';
-import { KIBANA_SETTINGS_TYPE } from '../../../common/constants';
+import { CLUSTER_ALERTS_ADDRESS_CONFIG_KEY, KIBANA_SETTINGS_TYPE } from '../../../common/constants';
+
+let loggedDeprecationWarning = false;
+
+export function resetDeprecationWarning() {
+  loggedDeprecationWarning = false;
+}
 
 /*
  * Check if Cluster Alert email notifications is enabled in config
  * If so, use uiSettings API to fetch the X-Pack default admin email
  */
-export async function getDefaultAdminEmail(config, callCluster) {
+export async function getDefaultAdminEmail(config, callCluster, log) {
   if (!config.get('xpack.monitoring.cluster_alerts.email_notifications.enabled')) {
     return null;
   }
 
+  const emailAddressConfigKey = `xpack.monitoring.${CLUSTER_ALERTS_ADDRESS_CONFIG_KEY}`;
+  const configuredEmailAddress = config.get(emailAddressConfigKey);
+
+  if (configuredEmailAddress) {
+    return configuredEmailAddress;
+  }
+
+  // DEPRECATED (Remove below in 7.0): If an email address is not configured in kibana.yml, then fallback to xpack:defaultAdminEmail
+  if (!loggedDeprecationWarning) {
+    const message = (
+      `Monitoring is using ${XPACK_DEFAULT_ADMIN_EMAIL_UI_SETTING} for cluster alert notifications,` +
```

Comment:
we need a space at end end of this line's string, so `notifications, which` has a space between